model/riscv_sys_regs.sail

```diff
@@ -186,8 +186,9 @@ function haveZbb()  -> bool = true
 function haveZbc()  -> bool = true
 function haveZbs()  -> bool = true
 
-/* BitManip Cryptography extension support. */
+/* BitManip Cryptography extensions support. */
 function haveZbkb() -> bool = true
+function haveZbkc() -> bool = true
 
 /* Cryptography Crossbar permutation instructions support. */
```

Comment:
Why does Zbkx get its own header but Zbkc doesn't? I'd be inclined to just have the one header for all three, we don't break down the exact meaning of Zb[abcs] and Zk* here